tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -557,3 +666,20 @@ func verifyIBUWorkloadReachable() {
 
 	Expect(err).NotTo(HaveOccurred(), "error reaching ibu workload")
 }
+
+func findInstalledCSV(expectedCSV string) bool {
+	csvList, err := olm.ListClusterServiceVersionInAllNamespaces(APIClient)
+	Expect(err).NotTo(HaveOccurred(), "error retrieving the list of CSV")
+
+	var installedCSV []string
+
+	for _, csv := range csvList {
+		if !slices.Contains(installedCSV, csv.Object.Name) {
```

Comment:
It's unclear to me what this is actually needed for

Reply:
to skip on found csv?

Reply:
not needed I think (copy/pasted)

Reply:
removed.